ldm/invoke/concepts_lib.py

```diff
@@ -115,10 +133,20 @@ def do_replace(match)->str:
         return self.match_concept.sub(do_replace, prompt)
 
     def get_concept_file(self, concept_name:str, file_name:str='learned_embeds.bin' , local_only:bool=False)->str:
-        if not self.concept_is_downloaded(concept_name) and not local_only:
+        if not self.concept_is_downloaded(concept_name) and not self.concept_is_local(concept_name) and not local_only:
```

Comment:
I think this would be more understandable as:
```if not (self.concept_is_downloaded(concept_name) or self.concept_is_local(concept_name) or local_only)```